src/modules/awaitAnimations.ts

```diff
@@ -76,16 +76,12 @@ function awaitAnimationsOnElement(element: Element): Promise<void> | false {
 			resolve();
 		};
 
-		const onEnd: EventListener = (event) => {
+		const onEnd = (event: TransitionEvent | AnimationEvent) => {
 			// Skip transitions on child elements
 			if (event.target !== element) {
 				return;
 			}
 
-			if (!isTransitionOrAnimationEvent(event)) {
```

Comment:
This check was only introduced to make the compiler happy, it's no longer required since we now have strict types for the event handler arguments